### ISSUE TITLE
tp.file.create_new strange behaviour since 1.16.1

### ISSUE THREAD

**Issue by piijai** [SilentVoid13/Templater#1216]
**Plugin information (please complete the following information):**

-   OS: macOS Ventura 13.6
-   Templater version: 1.16.3
-   Obsidian version: 1.4.14
-   Templater settings: Template folder location: "templates", Syntay Highlighting: on, Trigger Templater on new file creation: on, (add the templates not working with their associated command)]

**Describe the bug**
Quite similar bug to https://github.com/SilentVoid13/Templater/issues/1131, but it works with Templater Version 1.16.0.

With Templater Version 1.16.1 and above:
When I trigger the template, the note is created and it opens, the content from the content template appears for a moment and disappears instantly. 

With Templater Version 1.16.0: 
When I trigger the template, the note is created and it opens and the content appears in the file and stays.

With all Templater versions:
If the note already exists, the note gets opened.

You find my Template files below.

**Expected behavior**
That the journal note gets created with the content in it.

**Additional context**

Journal Week Next Template:
```
<%* 
const template = tp.file.find_tfile("Journal Week Next Template Content");

const desiredWeekday = 1; // Monday
const currentWeekday = moment().isoWeekday();
const missingDays = ((desiredWeekday - currentWeekday) + 7) % 7;
const nextMonday = moment().add(missingDays, "days");

const filename = moment(nextMonday).format("GGGG-MM-[KW]WW");
const folderPath = "1_areas/Journal";
const folder = this.app.vault.getAbstractFileByPath(folderPath);
const exists = await tp.file.exists(folderPath+ '/' + filename + '.md');
if (exists) {
await this.app.workspace.activeLeaf.openFile(this.app.vault.getAbstractFileByPath(folderPath + '/' + filename + '.md'));
} else {
await tp.file.create_new(template, filename, true, folder);
} %>
```

Journal Week Next Template Content
```
---
<%*
const desiredWeekday = 1; // Monday
const currentWeekday = moment().isoWeekday();
const missingDays = ((desiredWeekday - currentWeekday) + 7) % 7;
const nextMonday = moment().add(missingDays, 'days');

var fileDate = moment(nextMonday).format('YYYY-MM-DD');

// moment dates are mutable
let fileWeek = moment(fileDate).format('GGGG-[KW]WW');
let tagsDate = moment(fileDate).format("YYYY/MM/[KW]WW");
let yearLink = moment(fileDate).add(1, 'week').format('YYYY');
let monthLink = moment(fileDate).add(1, 'week').format('YYYY-MM');
let weekLink = moment(fileDate).add(1, 'week').format('GGGG-[KW]WW');
let prevWeek = moment(fileDate).subtract(1, 'week').format('GGGG-MM-[KW]WW');
let nextWeek = moment(fileDate).add(1, 'week').format('GGGG-MM-[KW]WW');
let prevWeekText = moment(fileDate).subtract(1, 'week').format('GGGG-[KW]WW');
let nextWeekText = moment(fileDate).add(1, 'week').format('GGGG-[KW]WW'); -%>
title: <% fileWeek %>
aliases: <% fileWeek %>
type: journal
date: <% fileDate %>
tags: journal/<% tagsDate %>
created: <% tp.date.now("YYYY-MM-DD HH:mm") %>
updated: <% tp.file.last_modified_date("YYYY-MM-DD HH:mm") %>
---

[[Journal]]

<%*
// ❮❮ ⋮ 2021-KW51 | 2021 › 2021-12 | 2021-KW52 ⋮ ❯❯
// [[path/to/file|display_text]]
let navStr = `❮❮ ⋮ [[1_areas/Journal/${prevWeek}|${prevWeekText}]] | [[1_areas/Journal/${yearLink}|${yearLink}]] › [[1_areas/Journal/${monthLink}|${monthLink}]] | [[1_areas/Journal/${nextWeek}|${nextWeekText}]] ⋮ ❯❯`;
tR += navStr
%>


# <% fileWeek %>

- [ ] 


## Termine

- [ ] 


## Next Weeks

- [ ] 


## Leseliste

- [ ] 


```

**Comment by Zachatoo** [comment]
I can't reproduce this issue with your templates, though you don't have steps so I wasn't sure how you were applying the template.

Does it happen in the sandbox vault with only the Templater plugin installed? It may be another plugin conflicting with Templater and trying to modify newly created files.

https://github.com/SilentVoid13/Templater/assets/6936914/f64ac557-79c5-4333-9d64-08d5c5d9bccb

**Comment by piijai** [comment]
Thanks @Zachatoo for testing.
With your hint, to use a sandbox vault, I finally found my mistake. I forgot about one setting in Templater. 
I had set up the "Journal Week Next Template" as a folder template for the folder "1_areas/Journal". 
So I trigger the same template with the Templater modal, that is also set as a folder template. Perhaps my template runs twice now. That would explain the empty file. And I don't see any errors in the console.

With the Templater version 1.16.0 and before, it didn't clear my template. 

When I removed the folder template, all is working with my template. So, it's all good for me. 
I don't know if you have defined a scenario how you want Templater to handle, if the triggered template is the same as the folder template. Perhaps my template is also a special case.

Again, thanks for your help.
